lib/postgres_ext/arel/5.0/visitors/postgresql.rb

```diff
@@ -0,0 +1,61 @@
+require 'arel/visitors/postgresql'
```

Comment:
This file looks like a copy of the 4.2 version. Is there are any reason to not just use that one, or are you still working on modifying it for Arel 7?